src/SearchButton.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const SearchButton = () => {
+  return (
+    <div>
```

Comment:
Why do you have a div around this button? It works, but I don't think it's necessary... Can you talk through why you added it?

Reply:
ya I will take it out